how-to/backups/install-gitolite.md

```diff
@@ -1,88 +1,148 @@
 (install-gitolite)=
 # How to install and configure gitolite
 
-{term}`Gitolite` provides a traditional source control management server for git, with multiple users and access rights management. 
```

Comment:
suggestion:

while touching it - should https://documentation.ubuntu.com/server/reference/glossary/#term-gitolite get a back link to this page?